files/ru/web/javascript/guide/indexed_collections/index.md

```diff
@@ -463,7 +463,7 @@ Array.prototype.forEach.call("a string", function(chr) {
 
 Для достижения максимальной гибкости и производительности, реализация типизированных массивов в JavaScript разделена на **буферы** и **представления**. Буфер ({{jsxref("ArrayBuffer")}}) это объект, представляющий из себя блок данных; он не имеет формата и не предоставляет возможности доступа к своему содержимому. Для доступа к памяти буфера вам нужно использовать представление. Представление являет собой контекст, имеющий тип данных, начальную позицию в буфере, и количество элементов — это позволяет представить данные в виде актуального типизированного массива.
 
-![Typed arrays in an ArrayBuffer](https://mdn.mozillademos.org/files/8629/typed_arrays.png)
+![Typed arrays in an ArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays/typed_arrays.png)
```

Comment:
The English version no longer has this image here, so it would be better to remove it from this page:

https://github.com/mdn/content/commit/42816e8a99c3ccd7bd6150adbfe886a7c025802f

Reply:
Thank you